src/pages/style-guide.md

```diff
@@ -0,0 +1,556 @@
+
+
+## Markdown formatting
+The rules in this section apply only to the main **documentation** site.
+
+### Headings
+- Use only heading levels 1, 2 and 3
+- The title of the page should have header level 1 (#)
+- Subheadings should be H2 (##), with any sections under that as H3 (###)
+- Do not use smaller headings as they look confusingly like normal text, and aren't included in the table of contents
+
+### Admonitions
+- Highlight blocks of text using the built-in [Docusaurus admonitions](https://docusaurus.io/docs/markdown-features/admonitions) feature
+- Use admonitions sparingly; having multiple within one page is overwhelming
+- Set a custom heading where possible
+
+   | ✅                            | ❌         |
+   | ---------------------------- | --------- |
+   | `:::info Function signature` | `:::info` |
+
+- Use "Tip" blocks to encourage readers to take action
+
+    | ✅                                                                                                                                |
+    | -------------------------------------------------------------------------------------------------------------------------------- |
+    | :::tip Avoiding duplicate column names<br></br>Make sure you include a prefix value to avoid duplicate column names.<br></br>::: |
+
+- Use "Note" or "Info" blocks to highlight information that doesn't necessarily require action, such as that about a newer version being available, how a specific component works, or what versions are supported
+- The "Warning" and "Danger" blocks are reserved for information about potential data loss or permissions breaches
+
+### Bold and italic
+- Highlight specific key words or phrases using bold (`**`, preferred) or italic (`_`)
+- Choose bold _or_ italic, don't combine them for the same words, or within the same sentence
+- Don't use bold or italic for components such as buttons or page titles (use quotation marks instead), or entire paragraphs
+
+   | ✅                                                    | ❌                                                      |
+   | ---------------------------------------------------- | ------------------------------------------------------ |
+   | events are **automatically tracked** once configured | **events** are _automatically tracked_ once configured |
+   | click on the "Add action" button                     | click on the _**Add action**_ button                   |
+
+### Code
+- Use inline code (single backticks, <code>`</code>) for field names, classes, file names, etc.
+
+    | ✅                             |
+    | ----------------------------- |
+    | a single `geo_location` field |
+    | a `event_id` UUID string      |
+    | tracking a `ScreenView` event |
+
+- Use code blocks (triple backticks, <code>```</code>) for code examples
+  - Specify the language next to the opening backticks, so that the code block is rendered correctly
```

Comment:
If this doesn't render appropriately, the language might need to be added to [this config](https://github.com/snowplow/documentation/blob/cfa199351c1e2b86ff1c70d65bdf8a7009a8d976/docusaurus.config.js#L157-L178)